pydirl/files_utils.py

```diff
@@ -31,9 +31,12 @@ def get_mtime(path):
     return os.path.getmtime(path)
 
 
-def directory_to_zipstream(path):
+def directory_to_zipstream(path, exclude=None):
```

Comment:
Minor concern: for modularity I would say we shouldn't expect the exclude parameter to be a regexp already compiled.